src/handlers/ban.ts

```diff
@@ -12,81 +12,83 @@ module.exports = {
+            color: Colors.Red,
+            author: {
+                name: `[BANNED] ${msg.user.tag}`,
+                iconURL: msg.user.displayAvatarURL(),
+            },
+            fields: [
+                {
+                    name: 'Member',
+                    value: msg.user.tag,
+                },
+                {
+                    name: 'Moderator',
+                    value: 'Unavailable - Audit log incomplete',
+                },
+                {
+                    name: 'Reason',
+                    value: 'Unavailable - Audit log incomplete',
+                },
+            ],
+            footer: { text: `User ID: ${msg.user.id}` },
+        });
+
+        if (modLogsChannel && !ModLogsExclude.some((e) => e)) {
+            if (!banLog) await modLogsChannel.send({ embeds: [noLogEmbed] });
+        }
+
         if (modLogsChannel && !ModLogsExclude.some((e) => e === executor.id)) {
             if (target.id === msg.user.id) {
-                const userBannedEmbed = makeEmbed({
-                    color: Colors.Red,
-                    author: {
-                        name: `[BANNED] ${msg.user.tag}`,
-                        iconURL: msg.user.displayAvatarURL(),
-                    },
-                    fields: [
-                        {
-                            name: 'Member',
-                            value: `${msg.user}`,
-                        },
-                        {
-                            name: 'Moderator',
-                            value: `${executor.id}`,
-                        },
-                        {
-                            name: 'Reason',
-                            value: `\u200B${reason}`,
-                        },
-                    ],
-                    footer: { text: `User ID: ${msg.user.id}` },
-                });
                 await modLogsChannel.send({ embeds: [userBannedEmbed] });
             } else {
-                const userBannedIncompleteEmbed = makeEmbed({
-                    color: Colors.Red,
-                    author: {
-                        name: `[BANNED] ${msg.user.tag}`,
-                        iconURL: msg.user.displayAvatarURL(),
-                    },
-                    fields: [
-                        {
-                            name: 'Member',
-                            value: msg.user.tag,
-                        },
-                        {
-                            name: 'Moderator',
-                            value: 'Unavailable - Audit log incomplete',
-                        },
-                        {
-                            name: 'Reason',
-                            value: 'Unavailable - Audit log incomplete',
-                        },
-                    ],
-                    footer: { text: `User ID: ${msg.user.id}` },
-                });
                 await modLogsChannel.send({ embeds: [userBannedIncompleteEmbed] });
```

Comment:
return instead of await

Reply:
It doesn't like a return here. (ESLint: Async arrow function 'executor' expected no return value. (consistent-return))